app/mailers/twilio_sender.rb

```diff
@@ -46,6 +46,8 @@ def self.get_phone_numbers_from_flow_execution(execution_id)
       Rails.logger.warn e.error_message
       return
     end
+    return if execution.nil? || execution.context.nil? || execution.context['trigger'].nil? || execution.context['trigger']['message'].nil?
```

Comment:
This is new syntax to me - seems kind of messier, no? Any reason this is better/worse?

Reply:
The intent of this method seems to set phone numbers to `nil` if they are not present in the message but uses improper syntax. What you proposed in this diff will provide two different return paths out of this function as it does not check for From/To nil condition. 

As for the syntax this is just standard safe navigation operator syntax. The reason it looks weird is because its not common to use the `[]` method with its arguments supplied via parameters.

Reply:
It has the same outcome (this method will return `nil` if it can not find valid phone numbers) but I'm fine with this change. Will be updating shortly.

Reply:
Updated here: 26670c3